.swiftlint.yml

```diff
@@ -43,6 +43,7 @@ reporter: "xcode" # reporter type (xcode, json, csv, checkstyle, junit)
 #   - mark
 #   - missing_docs
 #   - nesting
```

Comment:
I thought that we'd agreed to bump the parameter for nesting up to two or three, since the default value warns on two deep.